src/errors/fieldErrors.ts

```diff
@@ -18,7 +18,7 @@ export class FieldTypeInconsistencyError extends PushApiClientBaseError {
     });
 
     // TODO: CDX-844: display document with type inconsistencies
-    this.message +=
-      'Make sure to review and fix your document metadata before pushing';
+    this.message += `
+      Make sure to review and fix your document metadata before pushing`;
```

Comment:
🤨 why?

Reply:
For better readability.
Error message example:
```
Inconsistent types detected in the following field:
foo: STRING, DOUBLE
bar: STRING, DOUBLE, LONG
Make sure to review and fix your document metadata before pushing
```

Reply:
oh okay, mh tho here you will add the `      ` leading the Make sure message. Is that what you want 😕